src/Misc/XMLwrapper.cpp

```diff
@@ -32,9 +32,30 @@ namespace zyn {
 int  xml_k   = 0;
 bool verbose = false;
 
-const char *XMLwrapper_whitespace_callback(mxml_node_t *node, int where)
+#if MXML_MAJOR_VERSION <= 3
+// Define new types for the MXML3 types
```

Comment:
The phrasing of the comment makes it sound like you're defining mxml4 types that mirror mxml3 types, but it's the opposite, so maybe 
`//mimic datatypes present in mxml4 for compatibility` ?